assets/components/atoms/social-icon/social-icon.scss

```diff
@@ -0,0 +1,62 @@
+@charset 'utf-8';
+
+.social-icon {
+  display: inline-block;
+  padding: 0.6rem;
+  background: $gray-100;
+  border-radius: 0.17rem;
+  line-height: 0;
+
+  .icon {
+    top: 0;
+    font-size: 1.2rem;
+    fill: $white;
+  }
+}
+
+a.social-icon {
```

Comment:
You need to overide the `box-shadow` provide for all `a:focus, a.focus`

Reply:
I thought that as an accessibility mesure, we would keep focus on link the same everywhere. Do we just remove it, or find a corresponding focus style for each focusable component? 
And I should override it with another `!important` rule? Feels a bit dirty

Reply:
It's more a visual call to do, we have to check that with Guillaume. In the case you have to do it, you can approach it the [same way](https://github.com/epfl-idevelop/elements/commit/de009b381c1e6fe109b0769d42812f5daa4c59a6) I fix the event teaser.